core/cont/src/TClassTable.cxx

```diff
@@ -518,7 +533,7 @@ TClassRec *TClassTable::FindElementImpl(const char *cname, Bool_t insert)
 
 TClassRec *TClassTable::FindElement(const char *cname, Bool_t insert)
 {
-   if (!fgTable) return 0;
+   if (!CheckClassTableInit()) return nullptr;
```

Comment:
Return statement should go on a separate line, per coding conventions.

Reply:
Yes.  However, the surrounding code still follow the previous convention, so keeping it as is for now.